core/backend/src/BackendHubAccess.ts

```diff
@@ -188,18 +188,27 @@ export interface CreateNewIModelProps extends IModelNameArg {
  * Generally direct access to these methods should not be required, since higher-level apis are provided.
  * @note This interface is implemented in another repository. Any changes made to this interface must be validated against
  * the implementation found here: https://github.com/iTwin/imodels-clients/blob/main/itwin-platform-access/imodels-access-backend/src/BackendIModelsAccess.ts
- * @internal
+ * @public
```

Comment:
@pmconne I think public is good here, but it's possible that beta is more appropriate.

Reply:
Based on replies in the other comment, I'm going to mark this one as resolved: public is appropriate.